locales/en-US/app.l20n

```diff
@@ -95,3 +95,11 @@
 <tourDoneButton "Done">
 <userCountContainer "There are <span class=\"bold\">{{installation_count}}</span> people trying {{title}} right now!">
 <highlightPrivacy "Your privacy">
+
+// TODO: these strings are not currently localized.
```

Comment:
I don't think `//` is supported in L20N format for comments, just `/* ... */` syntax.